packages/cli/src/services/workflow-statistics.service.ts

```diff
@@ -10,6 +16,35 @@ import { TypedEmitter } from '@/typed-emitter';
 
 import { OwnershipService } from './ownership.service';
 
+const isStatusBillable = {
```

Comment:
Do we want to remove `billable` here?
```suggestion
const isStatusRootExecution = {
```